x-pack/plugins/lists/public/exceptions/components/builder/helpers.ts

```diff
@@ -96,7 +98,7 @@ export const filterExceptionItems = (
         return [...acc, item];
       } else if (createExceptionListItemSchema.is(item)) {
         // eslint-disable-next-line @typescript-eslint/no-unused-vars
-        const { meta: _, ...rest } = item;
+        const { meta, ...rest } = item;
```

Comment:
NOTE: @yctercero , this looked like it was fixed in the copied version so I removed it here. Let me know if that's wrong.

Reply:
Looks good! Thanks for the cleanup!